packages/frontend/pages/index.tsx

```diff
@@ -535,46 +557,39 @@ function TradePage() {
               <Header />
               <div className={classes.positionContainer}>
                 <SqueethInfo />
-                <PositionCard tradeCompleted={tradeCompleted} />
+                <PositionCard />
               </div>
             </div>
             <div className={classes.tradeDetails}>
-              {tradeType === TradeType.LONG ? (
-                <>
-                  <div>
-                    <LongChart />
-                  </div>
-                </>
-              ) : (
-                <>
-                  <div>
-                    <ShortChart vault={Vaults.Short} longAmount={0} showPercentage={true} setCustomLong={() => null} />
-                  </div>
-                </>
-              )}
+              <div className={tradeType === TradeType.LONG ? classes.displayBlock : classes.displayNone}>
+                <LongChart />
+              </div>
+
+              <div className={tradeType !== TradeType.LONG ? classes.displayBlock : classes.displayNone}>
+                <ShortChart vault={Vaults.Short} longAmount={0} showPercentage={true} setCustomLong={() => null} />
+              </div>
```

Comment:
I think it would be better to use conditional rendering than showing/hiding element for performance because it removes the element from DOM when the condition is not met

Reply:
Yes, I agree. nice catch